### ISSUE TITLE
Reversed signature?

### ISSUE THREAD

**Issue by matbesancon** [blegat/JuMP-dev_2019_tutorial#1]
I think I miss something from the semantics of `MOIB.added_constraint_types`.
In https://github.com/blegat/JuMP-dev_2019_tutorial/blob/master/Defining%20new%20bridges.ipynb
Code block 6:
```julia
function MOIB.added_constraint_types(::Type{NonnegativeBridge{T, F}}) where {T, F}
    return [(F, MOI.GreaterThan{T})]
end
```

That's a constraint that was already supported, shouldn't the return be `[(F, NonNegative)]`?

**Comment by blegat** [comment]
This indicates the list of constraints added by the bridges. This is used by `LazyBridgeOptimizer` to determine that in order to use the bridge, the optimizer should supports `F`-in-`MOI.GreaterThan{T}` constraints (or there must be a bridge for it)

**Comment by matbesancon** [comment]
Thanks. Also, there is a typo in the next block comment:
```julia
    # In the constructor, the function `f` of type `F` is passed to
    # `MOIU.add_scalar_constraint` which removes the constraint from `f` but
    # does not change its type so the type of the function in `MOI.GreaterThan`
    # will also be `F`.
```

constrant -> constraint

**Comment by blegat** [comment]
Thanks, fixed in e7464d55a2d081d5165e04bf126a2f5fb6c36364
